src/pages/public/movies/MovieDetailPage.vue

```diff
@@ -55,6 +67,11 @@ export default defineComponent({
         this.$router.push({ name: '404Page' });
       } finally {
         this.isLoading = false;
+
+        // But this line throws error: No manager or current instance at useMeta
+        //
+        // useMeta({ title: this.storedMovie.title });
```

Comment:
I'd say leave this for now until we touch upon composition API.
You are mixing composition API with options API and hence all the issues with this composable 🙏